### PR TITLE
free space probe added

### PR DESCRIPTION
**Pull request by FernandoGarzon**

### CODE REVIEW

File: cms/check_free_space [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+# - Fernando Garzon, <oscar.fernando.garzon.miguez@cern.ch>, 2021
+
+'''
+Probe to check the free space at each rse.
+'''
+
+import sys
+import traceback
+from rucio.core import monitor
+from rucio.core.rse import list_rses, get_rse_usage
```
Comment (ericvaandering) [inline cms/check_free_space:28]: If you can use rucio.api.rse functions that would be even better. There is often a 1:1 map.

File: cms/check_free_space [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            rucio_used = None
+            static_used = None
+            for usage in limits:
+                if usage['source'] == 'rucio':
+                    print('RUCIO_USED' , usage['used'])
+                    rucio_used = usage['used']
+                if usage['source'] == 'static':
+                    print('STATIC_USED' , usage['used'])
+                    static_used = usage['used']
+            if (rucio_used == None) or ( static_used == None ):
```
Comment (ericvaandering) [inline cms/check_free_space:47]: No spaces in the 2nd half of the line

Comment (ericvaandering) [reply cms/check_free_space:47]: Also comparisons with None should use "is" not "==" (research why).

**Review by dchristidis — COMMENTED** [review]
Some remarks from `pycodestyle`:
```
cms/check_free_space:40:54: E251 unexpected spaces around keyword / parameter equals
cms/check_free_space:40:56: E251 unexpected spaces around keyword / parameter equals
cms/check_free_space:45:39: E203 whitespace before ','
cms/check_free_space:48:40: E203 whitespace before ','
cms/check_free_space:50:41: E201 whitespace after '('
cms/check_free_space:50:61: E202 whitespace before ')'
cms/check_free_space:53:18: E111 indentation is not a multiple of four
cms/check_free_space:53:18: E117 over-indented
cms/check_free_space:55:62: E225 missing whitespace around operator
cms/check_free_space:56:5: E722 do not use bare 'except'
cms/check_free_space:57:14: E211 whitespace before '('
```

**Review by ericvaandering — CHANGES_REQUESTED** [review]
Same comment here. Please use Prometheus

File: cms/check_free_space [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                    rucio_used = usage['used']
+                if usage['source'] == 'static':
+                    print('STATIC_USED', usage['used'])
+                    static_used = usage['used']
+            if (rucio_used is None) or (static_used is None):
+                free_space = None
+            else:
+                free_space = int(static_used) - int(rucio_used)
+                print(rse['rse'], free_space)
+                monitor.record_gauge(stat='judge.free_space.%s' % rse['rse'], value=free_space)
```
Comment (ericvaandering) [inline cms/check_free_space:54]: You still are not using prometheus as requested. Why not?

**Review by ericvaandering — CHANGES_REQUESTED** [review]

**Pull request merged**